docs/shared/online-boutique/virtual-service.yaml

```diff
@@ -0,0 +1,29 @@
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: frontend
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - asm-ingress/asm-ingressgateway
```

Comment:
**Thought:**
Do we absolutely need to restrict this `VirtualService` to the `asm-ingressgateway` in the `asm-ingress` namespace?
I think it's fine to keep it. Yes, removing it would be simpler. But I don't foresee tutorials not using `asm-ingressgateway` in the `asm-ingress` namespace.

Reply:
I think it's better be more consistent. My plan here was that if we have more structured setup, it would hopefully create more consistency in we can write content in the docs